esp/src/src/ESPPackageProcess.ts

```diff
@@ -1,135 +1,21 @@
-        if (options.query.Target)
-            request['Target'] = options.query.Target;
-        if (options.query.Process)
-            request['Process'] = options.query.Process;
-        if (options.query.ProcessFilter)
-            request['ProcessFilter'] = options.query.ProcessFilter;
-
-        var results = WsPackageMaps.PackageMapQuery({
-            request: request
-        });
-
-        var deferredResults = new Deferred();
-        deferredResults.total = results.then(function (response) {
-            if (lang.exists("ListPackagesResponse.NumPackages", response)) {
-                return response.ListPackagesResponse.NumPackages;
-            }
-            return 0;
-        });
-        var context = this;
-        Deferred.when(results, function (response) {
-            var packageMaps = [];
-            for (var key in context._watched) {
-                context._watched[key].unwatch();
-            }
-            this._watched = {};
-            if (lang.exists("ListPackagesResponse.PackageMapList.PackageListMapData", response)) {
-                arrayUtil.forEach(response.ListPackagesResponse.PackageMapList.PackageListMapData, function (item, index) {
-                    var packageMap = context.get(item.Id);
-                    packageMap.updateData(item);
-                    packageMaps.push(packageMap);
-                    context._watched[packageMap.Id] = packageMap.watch("__hpcc_changedCount", function (name, oldValue, newValue) {
-                        if (oldValue !== newValue) {
-                            context.notify(packageMap, packageMap.Id);
-                        }
-                    });
-                });
-                if (options.sort) {
-                    packageMaps = context.sortPackageMaps(packageMaps, options.sort[0]);
-                }
-            }
-            deferredResults.resolve(packageMaps);
-        });
-
-        return QueryResults(deferredResults);
-    }
+import "dojo/i18n";
+// @ts-ignore
+import * as nlsHPCC from "dojo/i18n!hpcc/nls/hpcc";
+import * as ESPRequest from "./ESPRequest";
+
+var Store = declare([ESPRequest.Store], {
+    i18n: nlsHPCC,
+    service: "WsPackageProcess",
+    action: "ListPackages",
+    responseQualifier: "ListPackagesResponse.PackageMapList.PackageListMapData",
+    responseTotalQualifier: "ListPackagesResponse.PackageMapList.PackageListMapData",
+    idProperty: "__hpcc_id"
+    // startProperty: "PageStartFrom",
+    // countProperty: "PageSize",
 });
-
-var packageMap = declare([ESPUtil.Singleton], {
-    constructor: ESPUtil.override(function (inherited, args) {
-        inherited();
-        declare.safeMixin(this, args);
-        this.packageMap = this;
-    })
-});
-
 export function CreatePackageMapQueryObjectStore(options) {
     var store = new Store(options);
-    store = Observable(store);
-    var objStore = new ObjectStore({ objectStore: store });
-    return objStore;
-}
+    return new Observable(store);
```

Comment:
I don't think it matters, but I think its meant to be:  `return Observable(store);`